docs/tooling.rst

```diff
@@ -229,6 +230,36 @@ A new S3Compliant profile can be created using the s3compliant subcommand
   secret 's3-secret-chst2' created
   profile 's3-profile-5mmkj' created
 
+
+Kopia Repository Server creation using ``kanctl create``
+
+.. code-block:: bash
+
+    $ kanctl create repository-server --help
+    Create a new RepositoryServer
+
+    Usage:
+      kanctl create repository-server [flags]
+
+    Flags:
+      -h, --help                                           help for repository-server
+      -c, --locationCredsSecret string                     name of the secret containing kopia repository storage credentials
+      -l, --locationSecret string                          name of the secret containing kopia repository storage location details
+      -p, --prefix string                                  prefix to be set in kopia repository
+      -r, --repositoryPasswordSecret string                name of the secret containing password for the kopia repository
+      -z, --repositoryServerAdminUser string               kopia repository server admin user name
+      -a, --repositoryServerAdminUserAccessSecret string   name of the secret having admin credentials to connect to connect to kopia repository server
+      -u, --repositoryServerUser string                    name of the user to be created for the kopia repository server
```

Comment:
Shouldnt this be repositoryUser? Since we are using it to connect to repository and not the server